iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -228,26 +228,42 @@ public static void checkForecastDistance(Validable validable, int forecastDistan
         }
     }
 
-    public static void checkbPerSection(Validable validable, double bPerSection) {
-        if (Double.isNaN(bPerSection)) {
-            throw new ValidationException(validable, "susceptance per section is invalid");
+    public static void checkSectionNumber(Validable validable, int sectionNum) {
+        if (sectionNum < 0) {
+            throw new ValidationException(validable,
+                    "this number of section (" + sectionNum
+                            + ") should be greater than or equal to 0");
         }
+    }
+
+    public static void checkbPerSection(Validable validable, double bPerSection) {
```

Comment:
I fall in this exception on some TSOs, included for our French network !

Reply:
Corrected in CGMES conversion